docs/snippets/react/list-story-reuse-data.js.mdx

```diff
@@ -3,14 +3,19 @@
 
 import React from 'react';
 
+import List from './List';
+import ListItem from './ListItem';
```

Comment:
```suggestion
import { List } from './List';
import { ListItem } from './ListItem';
```

Reply:
I'll leave it as it is if you don't mind, as per the comment above.